tests/poms/MainDashboard/MainDashboard.ts

```diff
@@ -0,0 +1,15 @@
+import { Page, expect } from "@playwright/test";
+
+export default class MainDashboard {
+    readonly pageTitle = this.page.locator('.oxd-topbar-header-breadcrumb');
+    readonly sidePanel = this.page.locator('.oxd-sidepanel');
+    readonly hideSidePanelBtn = this.page.locator('[role="none"]');
+
+
+    constructor(private page: Page){};
+
+    async pageLoaded(){
+        await this.hideSidePanelBtn.waitFor();
+        await expect(this.pageTitle).toHaveText('Dashboard');
```

Comment:
Its hard to say if it is good idea to use expect inside PO but in general it is treated as bad habit.

Reply:
fixed